docs/testrunner-toolkit.md

```diff
@@ -178,7 +195,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |Puppeteer Version|Supported Browsers|
 |-----|----|
-|10.1.0|Please see [release notes](https://github.com/saucelabs/sauce-puppeteer-runner/releases/tag/v1.3.0)|
+|10.1.0|Please see [release notes](https://github.com/saucelabs/sauce-puppeteer-runner/releases/tag/v1.4.0)|
```

Comment:
Hey @mhan83 -- I threw in this update to match the one you added on the Puppeteer page? Let me know if that is correct. Thanks for doing this!

Reply:
Good catch, thank you!